apps/expo/src/app/_layout.tsx

```diff
@@ -1,12 +1,13 @@
+import "@bacons/text-decoder/install";
```

Comment:
added cause the metro engine in Expo 51 has TextEncoder built in but not TextDecoder